Unix/configure

```diff
@@ -2298,6 +2300,12 @@ then
     cp $rootrelative/etc/omigen.conf \$destdir/$sysconfdir/omigen.conf
 fi
 
+
+if test ! -f "\$destdir/$sysconfdir/omireg.conf"
```

Comment:
Both of these commands are in text that is output from this file, configure, to config.mak.  That file has two separate sections for the copy, one for dev that does and unconditional copy but uses -p to preserve times, while the other only adds if the old wasn't there.  These are config files, so the assumption is that users may well have changed them after the initial install and we don't mess with them on upgrade.